llmc/adamw.cuh

```diff
@@ -61,6 +61,16 @@ __global__ void adamw_kernel3(Tp* params_memory, float* master_params_memory, Tg
                  );
 }
 
+template <typename Tp>
+__global__ void init_from_master_kernel(Tp* params_memory, float* master_params_memory, size_t num_parameters,
+                                          ptrdiff_t w_stride, ptrdiff_t s_stride, unsigned int seed, bool check_identical) {
```

Comment:
unused `check_identical`?

Reply:
oops, that used to be for debug lots that I removed, thanks